docassemble/ALAutomatedTestingTests/data/questions/all_tests.yml

```
@@ -278,6 +279,15 @@ action buttons:
     action: end
     icon: laugh-wink
 ---
+id: Simple dropdown field
+question: Simple dropdown field
+field: simple_dropdown_field
```

Same as the other comment. You can batch these changes.

```suggestion
id: Single dropdown field
question: Single dropdown field
field: single_dropdown_field
```